src/lsp/cobol_parser/post/gen_post_actions.ml

```diff
@@ -141,7 +141,7 @@ let pp_production_posts ppf =
       | func -> Fmt.pf ppf "%s (@[<2>(%a)@ value@])" tag Fmt.text func
     in
     match post_action (last_item_attrs p),
-          post_action (Production.attributes p) with
+          post_action (Nonterminal.attributes (Production.lhs p)) with
```

Comment:
Could you check the generated file is the same with this change?

Reply:
The generated files are exactly the same